_config.yml

```diff
@@ -1,4 +1,4 @@
-title: "Pod 22.PX.X"
+title: "Pod 23.APR.1"
```

Comment:
The exact pod name would be "23.APR.PREP.1" :)
(Sorry I missed your question in the issue comments)

Reply:
Oh right. On it!